### DISCUSSION TITLE
A patch release for #600?

### DISCUSSION THREAD

**Discussion by ColinWttt** [Pagefind/pagefind#622] [Q&A]
It seems that #600 has resolved the segmentation for extended language, so no longer need to preprocess the search input using jieba-rs. Shall we release a new patch version?

**Comment by bglw** [comment]
🔜 

Aiming to do one within a couple of weeks to roll a few more changes in (and to avoid doing too many patch releases since they create work for the downstream projects that repackage pagefind) — anyone affected right now can use the latest alpha release which doesn't contain anything wild 🙂
